### PR TITLE
Fix Layout/LineLength - Bundle 2

### PR DESCRIPTION
**Pull request by macanudo527**
#### Second Bundle of Layout/LineLength Rubocop Offense Fixes

- Helps #6055 
- This is a bundle of fixes that were originally attached to #10714 

This makes code easier to read on smaller screens by reducing the line length. These fixes are bundled in groups of 60 to make it easier to review.

#### Release notes

Changelog Category: Technical changes

The title of the pull request will be included in the release notes.

### CODE REVIEW

File: spec/controllers/admin/subscription_line_items_controller_spec.rb
Comment (mkllnk) [inline spec/controllers/admin/subscription_line_items_controller_spec.rb:102]: This isn't pretty either. We could exclude `it` from this rule. Or we reword the test descriptions. Maybe this rule will make people use shorter descriptions in the future. :wink:

**Review by mkllnk — APPROVED** [review]
Nice, thank you.

File: spec/controllers/admin/subscription_line_items_controller_spec.rb
Comment (macanudo527) [reply spec/controllers/admin/subscription_line_items_controller_spec.rb:102]: This fits:
`it "returns serialized subscrip. line item w/ price estimate, based on variant" do`

Comment (mkllnk) [reply spec/controllers/admin/subscription_line_items_controller_spec.rb:102]: You don't need to change it now. I approved the pull request. 

The comment was about these changes in general, not just this one line. The `it` calls can be wordy and line breaks in strings are ugly. But I'm not sure what the solution is.

Comment (macanudo527) [reply spec/controllers/admin/subscription_line_items_controller_spec.rb:102]: I tried where possible to shorten them without losing the meaning. Should I go through them again and try to make them shorter?

Comment (mkllnk) [reply spec/controllers/admin/subscription_line_items_controller_spec.rb:102]: No, just leave it. We'll wait for a second review and can merge then.

File: spec/controllers/shops_controller_spec.rb
Comment (dacook) [inline spec/controllers/shops_controller_spec.rb:47]: TIL about "Free-spacing" mode in regex, provided by the `x` flag.

I think I would have instead simply `# rubocop:disable`d this one line. Or if writing from scratch perhaps use `hash_including()`, or just hardcode the ID in record.

Anyway this is fine :)

**Review by dacook — APPROVED** [review]
Great, thanks for working through these

**Comment by dacook** [comment]
This only updates specs (thanks for chunking it this way), so we don't need any manual testing. Ready to merge!

File: spec/controllers/shops_controller_spec.rb
Comment (macanudo527) [reply spec/controllers/shops_controller_spec.rb:47]: Oh come on. Using `# rubocop: disable` is cheating, where is the fun in that?:)

Thanks for the heads up on `hash_including()`!

Comment (dacook) [reply spec/controllers/shops_controller_spec.rb:47]: 😆

**Pull request merged**
